src/canmatrix/xls.py

```diff
@@ -309,6 +309,28 @@ def dump(db, file, **options):
     # save file
     workbook.save(file)
 
+############################ load ###############################
+
+def parse_value_name_collumn(value_name, value, signal_size, float_factory):
```

Comment:
```suggestion
def parse_value_name_column(value_name, value, signal_size, float_factory):
```

Reply:
Search and replace `collumn` everywhere with `column`.

https://en.wiktionary.org/wiki/column

Reply:
thank you for this hint, my spelling is not the best, ...